goodreads-predictor/src/goodreads_predictor/pipelines/feature_engineering/pipeline.py

```diff
@@ -5,9 +5,13 @@
 
 from kedro.pipeline import Pipeline, pipeline, node
 from .nodes import apply_book_attributes
+from .nodes import merge_description_embeddings
+from .nodes import perform_clustering_analysis
 
 
 def create_pipeline(**kwargs) -> Pipeline:
     return pipeline([
-        node(func=apply_book_attributes, inputs=['filtered_books'], outputs=['books_features', 'feature_cutoffs'], name='add_engineered_features')
+        node(func=apply_book_attributes, inputs=['filtered_books', 'description_embeddings'], outputs=['books_features', 'feature_cutoffs'], name='add_engineered_features'),
+        node(func=merge_description_embeddings, inputs=['books_features', 'description_embeddings'], outputs='merged_df', name='merge_description_embeddings'),
+        node(func=perform_clustering_analysis, inputs=['merged_df', 'n_clusters'], outputs='books_features_clustered', name='perform_clustering_analysis'),
```

Comment:
Where is n_clusters coming from? This currently fails when trying to run because it cannot resolve n_clusters. Ideally, this would be in conf/base/parameters_feature_engineering.yml and then referenced in the inputs list as `"params:n_clusters"`